go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -696,8 +700,16 @@ func main() {
 		}
 		fromSrc += " && eth.dst==" + srcPodInfo.StorMAC
 		fromSrc += " && eth.src==" + srcPodInfo.MAC
-		fromSrc += " && ip4.dst==" + dstSvcInfo.IP
-		fromSrc += " && ip4.src==" + srcPodInfo.IP
+		if dstSvcInfo.IsIPv4 {
```

Comment:
@mrVectorz but actually what I'd do is a small function like:

```
func getL3Ver(ip string) string {
    if net.ParseIP(ip).To4() != nil {
        return "ip4"
    }
    return "ip6"
}
```

and then the below all becomes:

`fromSrc += " && " + getL3Ver(dstSvcInfo.IP) + ".dst==" + dstSvcInfo.IP`

or even, perhaps clearer:

`fromSrc += fmt.Sprintf(" && %s.dst==%s", getL3Ver(dstSvcInfo.IP), dstSvcInfo.IP)`

Reply:
Sounds good, I'll make the adjustments